docs/index.rst

```diff
@@ -60,6 +39,19 @@ Extension Development Guide
     How to migrate from one version of Extension API to a new one.
 
 
+Custom Color Themes
+===================
+
+.. toctree::
+   :caption: Custom Color Themes
+   :hidden:
+
+   themes/themes
+
+:doc:`themes/themes`
+    Create your own color themes
```

Comment:
Related to this, I really think we should recommend people to use https://github.com/Ulauncher/Ulauncher/discussions/categories/themes instead of your gist (I just created this discussion).

I'm not sure what validation we want for it. We can test it here first: https://github.com/Ulauncher/Test-Discussions

Reply:
I like the discussions approach for themes. I want to create an automation for migrating them from the Gist. Post v6 release though.